CMakeLists.txt

```diff
@@ -23,18 +23,28 @@ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake;${CMAKE_MODULE_PATH}")
 
 set(XEUS_CPP_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/include)
 
+# xeus-cpp tag files
+set(XEUS_CPP_DATA_DIR "share/xeus-cpp" CACHE STRING "xeus-cpp data directory")
```

Comment:
Why do we need to put these in the cache?

Reply:
It was there in the previous code. CACHE just turns up in the cmake config tool as setting you can set. The initial commit for it was made by @mvassilev

Reply:
Ok. We can set them as normal variables.